core/whistle-1.0.0/src/api/wapi_notifications.erl

```diff
@@ -399,6 +399,8 @@ headers(<<"skel">>) ->
     ?SKEL_HEADERS ++ ?OPTIONAL_SKEL_HEADERS;
 headers(<<"topup">>) ->
     ?TOPUP_HEADERS ++ ?OPTIONAL_TOPUP_HEADERS;
+headers(<<"port_request">>) ->
```

Comment:
@macpie FYI, you forgot to add this clause in for the preview stuff to work. Please be sure to test both the Preview in the UI and the actual generating of the notification by the system.